skrub/datasets/_utils.py

```
@@ -1,5 +1,18 @@
+from os import environ
 from pathlib import Path
 
+from loguru import logger
+
+DATA_HOME_ENVAR_NAME = "SKRUB_DATA_DIRECTORY"
+DATA_HOME_ENVAR = environ.get(DATA_HOME_ENVAR_NAME)
+
+if DATA_HOME_ENVAR and (path := Path(DATA_HOME_ENVAR)).is_absolute():
```

when the variable is set but is a relative path we silently ignore it?

---

Yes, i supposed that the docstr + the log displaying the content of `DATA_HOME_DEFAULT` were enough explicit.
Would you like me to change it?